src/components/MBWay/MBWay.tsx

```diff
@@ -9,6 +9,10 @@ export class MBWayElement extends UIElement {
     private static type = 'mbway';
 
     formatProps(props: UIElementProps): UIElementProps {
+        if (props.data) {
+            props.data.email = props.data.shopperEmail || props.data.email;
+            props.data.phoneNumber = props.data.telephoneNumber || props.data.telephoneNumber;
```

Comment:
```suggestion
            props.data.phoneNumber = props.data.telephoneNumber || props.data.phoneNumber;
```

Was this the intended change?